source/database/managing-database.rst

```diff
@@ -89,13 +89,18 @@ likely that further users will need to be added over time.
 
 This can be done using the openstack commandline. Below we can see two example
 of how we can add a new user to our myDB database. One example creates a
-user that can access the database from any location. This is the same behaviour
+user that can access the database from any location. This is the same behavior
 that is displayed when the user is created as part of the initial database
 instance creation.
 
 The other example uses the ``--host`` argument which creates a user that can
 only connect from a specified IP address.
 
+.. Note::
+
+  Support for commands using "openstack database ``user`` or ``db``
```

Comment:
```
`openstack database user` and `openstack database db`
```

It's not `not been added`, it's `not supported` actually. I have provided an example in previous comment, please have a look.

Reply:
Acknowledged. I've rephrased the passage to make that clear.